tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -2850,15 +2857,12 @@ private String handleCurrency(String xpath, XPathParts parts, String value) {
         return null;
     }
 
-    private String handleDateRangePattern(String value) {
-        String result;
+    private void handleDateRangePattern(String value, List<String> examples) {
         SimpleDateFormat dateFormat = icuServiceBuilder.getDateFormat("gregorian", 2, 0);
-        result =
-                format(
-                        value,
-                        setBackground(dateFormat.format(DATE_SAMPLE)),
-                        setBackground(dateFormat.format(DATE_SAMPLE2)));
-        return result;
+        examples.add(format(
+            value,
+            setBackground(dateFormat.format(DATE_SAMPLE)),
+            setBackground(dateFormat.format(DATE_SAMPLE2))));
```

Comment:
instead of making this method void, how about making it call the single-arg version and returning that String value

Reply:
Right, it's adding to the 'examples' parameter